src/components/AdminDashboard/AdminDashboard.js

```diff
@@ -70,117 +68,76 @@ const AdminDashboard = () => {
     window.scrollTo(0, 0);
   }, [HAS_VISITED_BEFORE]);
 
+  // resets the selected incidents when switching from approved to unapproved page
+  useEffect(() => {
+    setSelectedIds([]);
+    setAllSelected(false);
+  }, [listType]);
+
   const modalHandler = () => {
     setShowModal(false);
   };
 
-  const lastPage = Math.ceil(unapprovedIncidents.length / incidentsPerPage);
-
   const oktaAxios = useOktaAxios();
 
-  // getting unapproved/pending incidents from the database
-  useEffect(() => {
-    getData(oktaAxios, setUnapprovedIncidents);
-  }, []);
+  const fetchIncidents = () => {
+    getApprovedIncidents(oktaAxios)
+      .then(res => setApprovedIncidents(res))
+      .catch(err => console.log(err));
 
-  // getting approved incidents from database
-  useEffect(() => {
-    oktaAxios
-      .get('/dashboard/incidents/approved')
-      .then(res => {
-        setIncidents(res.data);
-      })
-      .catch(err => {
-        console.log(err);
-      });
-  }, []);
+    getPendingIncidents(oktaAxios)
+      .then(res => setPendingIncidents(res))
+      .catch(err => console.log(err));
 
-  // getting form-responses from DS database
+    getFormResponses(oktaAxios)
+      .then(res => setFormResponses(res))
+      .catch(err => console.log(err));
+  };
+
+  // getting all incident data
   useEffect(() => {
-    axios
-      .get(
-        'http://hrf-bw-labs37-dev.eba-hz3uh94j.us-east-1.elasticbeanstalk.com/to-approve'
-      )
-      .then(res => {
-        setFormResponses(res.data);
-      })
-      .catch(err => {
-        console.log(err);
-      });
+    fetchIncidents();
   }, []);
```

Comment:
fetchIncidents sets the approved incidents, but this useEffect was setting form responses, do we just not need it to do that anymore because that component state isn't necessary?

Reply:
The form-response data is now fetched on line 92 (currently), called from the useEffect just below